ci-operator/step-registry/edge/ci/ocm-publish/edge-ci-ocm-publish-commands.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
+# Uncomment for Local testing
+#export IMAGE_NAME=assisted-service
+#export IMAGE_TAG=ocm-2.4
+#export IMAGE_VERSION=ocm-2.4
+#export IMAGE_REGISTRY=quay.io
+#export IMAGE_ORG=edge-infrastructure
+
+printf "\nAssisted Installer Image Info:\n"
+printf "IMAGE_NAME:     %s\n" ${IMAGE_NAME}
+printf "IMAGE_TAG:      %s\n" ${IMAGE_TAG}
+printf "IMAGE_VERSION:  %s\n" ${IMAGE_VERSION}
+printf "IMAGE_REGISTRY: %s\n" ${IMAGE_REGISTRY}
+printf "IMAGE_ORG:      %s\n" ${IMAGE_ORG}
+
+# Grab the most recent manifest digest of a tag
+digest=$(curl -G https://${IMAGE_REGISTRY}/api/v1/repository/${IMAGE_ORG}/${IMAGE_NAME}/tag/\?specificTag=${IMAGE_TAG} | \
+    jq -e -r '.tags[] | select((has("expiration") | not)) | .manifest_digest')
+
+# Uncomment to debug curl statement for manifest digest tag
+#curl -G https://${IMAGE_REGISTRY}/api/v1/repository/${IMAGE_ORG}/${IMAGE_NAME}/tag/\?specificTag=${IMAGE_TAG} | jq '.'
+
+# Fail if digest empty
+if [ -z ${digest} ]; then
+    echo "Unable to get remote image manifest. digest: ${digest}"
+    exit 1
+fi
+
+# Grab the git commit associated with this image
+vcsref=$(curl -G https://${IMAGE_REGISTRY}/api/v1/repository/${IMAGE_ORG}/${IMAGE_NAME}/manifest/${digest}/labels | \
+    jq -e -r '.[][] | select(.key | contains("vcs-ref")) .value')
+
+# Uncomment to debug curl statement for git commit
+#curl -G https://${IMAGE_REGISTRY}/api/v1/repository/${IMAGE_ORG}/${IMAGE_NAME}/manifest/${digest}/labels | jq '.'
+
+# Fail if vcsref is empty
+if [ -z ${vcsref} ]; then
+    echo "Unable to determine git commit. vcsref: ${vcsref}"
+    exit 1
+fi
+printf "Assisted Installer Image Info From Quay:\n"
+printf "Image Digest SHA: ${digest}\n"
+printf "Image Git (VCS) Ref: ${vcsref}\n"
+
+export OSCI_COMPONENT_NAME=${IMAGE_NAME}
+export OSCI_COMPONENT_TAG=${IMAGE_TAG}
+export OSCI_COMPONENT_VERSION=${IMAGE_VERSION}
+export OSCI_COMPONENT_SHA256=${vcsref}
+export OSCI_COMPONENT_REPO=${REPO_OWNER}/${REPO_NAME}
+export OSCI_IMAGE_REMOTE_REPO=${IMAGE_REGISTRY}/${IMAGE_ORG}
+export OSCI_PUBLISH_DELAY=0
+
+printf "OCM Pipeline Info:\n"
+printf "OSCI_PIPELINE_GIT_BRANCH: %s\n" ${OSCI_PIPELINE_GIT_BRANCH}
+printf "OSCI_COMPONENT_NAME: %s\n" ${OSCI_COMPONENT_NAME}
+printf "OSCI_COMPONENT_TAG: %s\n" ${OSCI_COMPONENT_TAG}
+printf "OSCI_COMPONENT_VERSION: %s\n" ${OSCI_COMPONENT_VERSION}
+printf "OSCI_COMPONENT_SHA256: %s\n" ${OSCI_COMPONENT_SHA256}
+printf "OSCI_COMPONENT_REPO: %s\n" ${OSCI_COMPONENT_REPO}
+printf "OSCI_IMAGE_REMOTE_REPO: %s\n" ${OSCI_IMAGE_REMOTE_REPO}
+
+
+#cd /opt/build-harness/build-harness-extensions/modules/osci/
+#make osci/publish BUILD_HARNESS_EXTENSIONS_PATH=/opt/build-harness/build-harness-extensions
+
+export OSCI_PIPELINE_PRODUCT_PREFIX ?= release
```

Comment:
Why not 
`export OSCI_PIPELINE_PRODUCT_PREFIX="{OSCI_PIPELINE_PRODUCT_PREFIX:-release}"` :thinking:

Reply:
It was a copy pasta from the OCM scripts.. going to do a different thing though. See: https://github.com/open-cluster-management/build-harness-extensions/tree/master/modules/osci for the scripts this is running.